### DISCUSSION TITLE
能否扩展一下get_group_info()接口返回的数据字段

### DISCUSSION THREAD

**Discussion by HuangShiJin1015** [howmanybots/onebot#91] [想法 / 建议]
我在调用get_group_info（）接口时，发现返回的数据指向只有四个，分别是：
group_id（群号），group_name（群名称） max_member_count（群规模）member_count（群成员个数）。

能否增加一下群简介、群创建时间、群头像这些数据字段项呢。

**Comment by stdrc** [comment]
确实可以，如果目前有实现能做的话，确实可以加到标准里

**Comment by HuangShiJin1015** [comment]
> 确实可以，如果目前有实现能做的话，确实可以加到标准里

嗯嗯，好的。我通过更改cqhttp的api包，新增了一些字段，发现可以返回群介绍和创建时间，群头像好像还是没办法获取到。
